tests/test_project.py

```diff
@@ -333,20 +406,6 @@ async def test_create_env_same_name(client):
     assert f"Project with id={project_id} already has an environment with name dev1" in result.result["message"]
 
 
-async def test_project_cascade(client):
-    result = await client.create_project("env-test")
-    project_id = result.result["project"]["id"]
-
-    result = await client.create_environment(project_id=project_id, name="dev")
-    result = await client.create_environment(project_id=project_id, name="prod")
-
-    result = await client.delete_project(project_id)
-    assert result.code == 200
-
-    result = await client.list_environments()
-    assert len(result.result["environments"]) == 0
```

Comment:
This test case also calls into the `list_environments()` endpoint after the deletion of the project. The new test cases don't do that. Make sure to be very careful when deleting tests, so that we don't lose test coverage.